postgres-driver/pocket_session.go

```diff
@@ -0,0 +1,15 @@
+package postgresdriver
+
+import (
+	"context"
+
+	"github.com/pokt-foundation/transaction-db/types"
+)
+
+func (d *PostgresDriver) WriteSession(ctx context.Context, session types.PocketSession) error {
```

Comment:
This PR is already merged, so obviously not a blocker: but I recommend keeping all the methods of a struct in the same file, rather than scattering them around in different files. The current style makes it challenging to identify different interfaces a type implements, or to see the entire struct and its behavior in one place.

If the struct, at some point, becomes large enough to warrant separating across code files, the proper method to address it would most likely be breaking the responsibilities across more than 1 struct and then compose them into a single driver/service.

Reply:
This was to keep the standard with PHD, and this files will grow with time. On PHD I started with just one file and I ended up dividing because of that